packages/tests/vendor/@types/testing-library__jest-dom/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for @testing-library/jest-dom 5.14
```

Comment:
We are currently using `@testing-library/jest-dom` version `5.16.5` vs `5.14` here, I guess we should update `matchers.d.ts` to use the same version?

Reply:
There were no API changes since `5.14` and those are the latest available types, so I left the version comment intact to make it easy to realize which version is vendored.